src/library/components/Button.tsx

```diff
@@ -33,7 +33,7 @@ const Button = forwardRef(({ children, variant, icon, iconAlignment = 'left', ic
   }
 
   if (icon && !children) {
-    return <button ref={ref} className={`bsds-button-icon ${classes} ${className || ''}`} {...buttonAttrs}><Icon name={icon} size={iconSize} /></button>;
+    return <button ref={ref} className={`bsds-button-icon ${classes} ${className || ''}`} {...buttonAttrs}><Icon name={icon} size={iconSize} className={className || ''}/></button>;
```

Comment:
Can we do `|| undefined` instead? That way if className is an empty sting, it won't add the class attribute to the component. If you pass className an empty string it renders `class=""`.

Reply:
Does the value of className get passed to both the button and the icon?